basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -785,6 +785,18 @@ public static void addGlobalData(Site site, Properties props, Properties custom,
 		M_log.debug("before custom="+custom);
 		LTI2Util.substituteCustom(custom, lti2subst);
 		M_log.debug("after custom="+custom);
+		
+		//TODO check if TII activated?
+		//substitute resources params
+		if(custom.getProperty(BasicLTIConstants.RESOURCE_LINK_ID) != null){
+			M_log.debug("we have custom link id " + custom.getProperty(BasicLTIConstants.RESOURCE_LINK_ID));
+			setProperty(ltiProps,BasicLTIConstants.RESOURCE_LINK_ID,custom.getProperty(BasicLTIConstants.RESOURCE_LINK_ID));
```

Comment:
Discussed on call, this is is because otherwise it gets the RESOURCE_LINK_ID from the placement and for all TII tools it would be the same.

Remaining question how the LTI tools work when you don't put them in a page.